utils/utils.py

```diff
@@ -733,10 +777,20 @@ def run_benchmarks(
         "extended_functionality": extended_functionality,
     }
 
-    etl_results = []
-    ml_results = []
+    db_params = DBParams(
+        server=db_server, port=db_port, user=db_user, password=db_pass, name=db_name
+    )
+    predefined_fields = {
+        "OmnisciCommitHash": commit_omnisci,
+        "OmniscriptsCommitHash": commit_omniscripts,
+        "ModinCommitHash": commit_modin,
+    }
+    etl_reporter = ResultReporter(db_params, db_table_etl, predefined_fields, ["t_connect"])
+    ml_reporter = ResultReporter(db_params, db_table_ml, predefined_fields, tuple())
```

Comment:
Why is it `tuple()` instead of `[]`?

Reply:
To have immutable data. But for consistency it's better to either have lists on line 788 and 789 or tuples.
```suggestion
    ml_reporter = ResultReporter(db_params, db_table_ml, predefined_fields, [])
```